lib/perl/Genome/Db/Ensembl/Command/Run/Base.pm

```diff
@@ -382,7 +382,6 @@ sub api {
     my $self = shift;
     return Genome::Db::Ensembl::Api->get_or_create(
         version => $self->ensembl_version,
-        test_name => $ENV{GENOME_SOFTWARE_RESULT_TEST_NAME},
```

Comment:
Would the `GENOME_ALIGNER_INDEX_TEST_NAME` variable be useful here?  That's stretching the name of it a bit, but that's the variable for foundational results that depend on only things like the reference.